bundle/manifests/rhtas-operator.clusterserviceversion.yaml

```
@@ -296,8 +296,8 @@ metadata:
         }
       ]
     capabilities: Seamless Upgrades
-    containerImage: registry.redhat.io/rhtas/rhtas-rhel9-operator@sha256:52ba6cd82bc400a08c6f89811e8086126596a873b9b12619de8c5064a2d4faf7
-    createdAt: "2024-11-07T15:59:23Z"
+    containerImage: registry.redhat.io/rhtas/rhtas-rhel9-operator-v1-1@sha256:862b8804b0896c0df65563ba6a1467d28edc3c72fb1ad1cff9c96cc8b6a9a4a0
```

```suggestion
    containerImage: registry.redhat.io/rhtas/rhtas-rhel9-operator@sha256:862b8804b0896c0df65563ba6a1467d28edc3c72fb1ad1cff9c96cc8b6a9a4a0
```

We dont need this change here :)